src/99-appendices/01-contributors.md

```diff
@@ -196,3 +196,4 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 -   Lyuba Zehl 📖
 -   Melanie Ganz-Benjaminsen 📖🔣💻🤔📆🔍📢
 -   Martin Noergaard 📖🔣💻🤔📢
+-   Eduard Ort 📖
```

Comment:
I am removing this entry, because you'll be added to the list as part of this PR: #736 

```suggestion
```